### PR TITLE
selinux spc test: fix CI breakage

### PR DESCRIPTION
**Pull request by edsantiago**
The SELinux spc test is breaking under some CI circumstances in
Fedora 30; most notably, a CI-built buildah (with type != bin_t)
is failing with:

       FAIL: container context matches our own
   expected: 'system_u:system_r:unconfined_service_t:s0'
     actual: 'system_u:system_r:spc_t:s0'

It is not clear to me what the initial purpose of this test
is, since (I believe) the container context should always
be spc_t regardless of the calling process context; likewise,
role should always be system_r.

This commit tweaks the test to check role/type/range
individually.

Needed for #2115

Signed-off-by: Ed Santiago <santiago@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
tests green

**Comment by cevich** [comment]
Testing along with bats fix in: #2115 (rebased on this pr)

**Comment by cevich** [comment]
> tests green

FYI- until #2115 goes in, that unfortunately doesn't mean much.  The Travis tests don't do anything real with selinux :(

**Comment by TomSweeneyRedHat** [comment]
LGTM, but want a head nod from @rhatdan

**Comment by cevich** [comment]
@rhatdan this is green on the "actually tested" side too (#2115).

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

File: tests/selinux.bats [hunk 1]
```diff
@@ -40,16 +40,23 @@ load helpers
...
-  fi
   # Create a container and read its context as a baseline.
   run_buildah from --quiet --security-opt label=disable --quiet --signature-policy ${TESTSDIR}/policy.json $image
   cid=$output
   run_buildah run $cid sh -c 'tr \\0 \\n < /proc/self/attr/current'
-  expect_output "$firstlabel" "container context matches our own"
+  context=$output
+
+  # Role and Type should always be constant. (We don't check user)
+  role=$(awk -F: '{print $2}' <<<$context)
```
[the remark this answers is not in the document]
Comment (edsantiago) [reply tests/selinux.bats]: Oh that's really useful, thank you!

**Pull request closed without merging**